### ISSUE TITLE
ASN1Integer.fromInt() creates wrong encoded bytes

### ISSUE THREAD

**Issue by dhenneke** [wstrange/asn1lib#24]
Consider the following test:
```dart
test('fromInt vs BigInt.from', () {
  var si = ASN1Integer.fromInt(1);
  var sb = ASN1Integer(BigInt.from(1));

  expect(si.encodedBytes, equals(sb.encodedBytes));
});
```

This fails with the following error:
```
Expected: [2, 1, 1]
  Actual: [0, 1, 1]
   Which: was <0> instead of <2> at location [0]

```

It looks like the tag is not set to INTEGER_TYPE (or 0x02) in the [`ANS1Integer.fromInt(int x)`](https://github.com/wstrange/asn1lib/blob/92fd317054b3f4b2c82b1911966caa97b529afb2/lib/asn1integer.dart#L15) constructor but is in the original constructor. This also breaks the [dartdap](https://github.com/wstrange/dartdap) package for me, since the generated frames are invalid and are not interpreted and answered by the ldap server.

The following code also crashes since it can't parse the bytes:
```dart
// correct
var seq0 = ASN1Sequence(tag: 48)..add(ASN1Integer(BigInt.from(1)));
ASN1Sequence.fromBytes(seq0.encodedBytes);

// crash
var seq1 = ASN1Sequence(tag: 48)..add(ASN1Integer.fromInt(1));
ASN1Sequence.fromBytes(seq1.encodedBytes);
```

**Comment by wstrange** [comment]
Thanks for catching this. I added your test, and verified the fix, and uploaded 0.5.3 to pub.

I will release a new Dartdap ASAP.

**Comment by dhenneke** [comment]
Awesome. Thanks for the quick fix 😊
